tests/trilinos/precondition_muelu_q_iso_q1.cc

```diff
@@ -276,7 +276,7 @@ void Step4<dim>::solve ()
     check_solver_within_range(
       solver.solve (system_matrix, solution, system_rhs,
                     preconditioner),
-      solver_control.last_step(), 25, 34);
```

Comment:
this must be related to 

> Fixed: IndexSet::make_trilinos_map now treats non-ascending but contiguous IndexSets correctly...

 right? Because other changes are only inside Asserts.